### PR TITLE
DIV-6199: Displaying different fees to solicitor depending on whether…

### PR DESCRIPTION
**Pull request by jvictorcf**
… the case is being amended or issued

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: definitions/divorce/json/CaseField/CaseField-solicitorDn-nonprod.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "ID": "LabelSolPaymentPara-1",
+    "Label": "Amount to pay: **£${solApplicationFeeInPounds}**",
```
Comment (yarpo) [inline definitions/divorce/json/CaseField/CaseField-solicitorDn-nonprod.json:6]: are you sure that we can add field to prod and then update it without data migration?

File: test/unit/definitions/divorce/CaseEventToFields.test.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ const { expect, assert } = require('chai');
 const { find } = require('lodash');
 
 const caseEvent = Object.assign(require('definitions/divorce/json/CaseEvent'), []);
-const caseField = Object.assign(require('definitions/divorce/json/CaseField/CaseField'), []);
+const caseField = [].concat(require('definitions/divorce/json/CaseField/CaseField.json')).concat(require('definitions/divorce/json/CaseField/CaseField-prod.json'));
```
Comment (yarpo) [inline test/unit/definitions/divorce/CaseEventToFields.test.js:6]: I suppose we should test here both options - non-prod and prod. 2 test cases.

File: definitions/divorce/json/CaseField/CaseField-solicitorDn-nonprod.json (hunk 1 above)
Comment (jvictorcf) [reply definitions/divorce/json/CaseField/CaseField-solicitorDn-nonprod.json:6]: This field's only utility is to show as a variable rate in the screen.
It's not important for the logic and it doesn't matter if it's empty.

File: test/unit/definitions/divorce/CaseEventToFields.test.js (hunk 2 above)
Comment (jvictorcf) [reply test/unit/definitions/divorce/CaseEventToFields.test.js:6]: In all the other test scenarios, we're only testing prod, right?

Comment (yarpo) [reply test/unit/definitions/divorce/CaseEventToFields.test.js:6]: I'm not so sure :)

```
describe('AuthorisationCaseEvent', () => {
  it('should contain a unique case type, case event ID and role (no duplicates) for non-prod', () => {
    const nonProd = mergeJsonFilesFor('nonprod');
    const uniqResult = uniqWith(nonProd, isDuplicated);

    expect(uniqResult).to.eql(nonProd);
  });

  it('should contain a unique case type, case event ID and role (no duplicates) for prod only', () => {
    const prodOnly = mergeJsonFilesFor('prod');
    const uniqResult = uniqWith(prodOnly, isDuplicated);

    expect(uniqResult).to.eql(prodOnly);
  });
});
```

div-ccd-definitions/test/unit/definitions/divorce/AuthorisationCaseEvent.test.js

**Pull request merged**